OpenEXR/IlmImfTest/testMultiPartSharedAttributes.cpp

```diff
@@ -152,7 +152,7 @@ testMultiPartOutputFileForExpectedFailure (const vector<Header> & headers,
     try
     {
         remove(fn.c_str());
-        MultiPartOutputFile file(fn.c_str(), &headers[0],headers.size());
+        MultiPartOutputFile file(fn.c_str(), (headers.size()>0 ? &headers[0] : NULL) , headers.size() );
```

Comment:
Wouldn't header.data() be a more proper way of accessing the data?

Reply:
Yes it would, but is vector::data() available in all C++ versions that are supported by OpenEXR?